thirdeye-ui/src/app/components/subscription-group-wizard/subscription-group-details/recipient-details/groups-editor/slack/slack.component.tsx

```diff
@@ -67,19 +73,33 @@ export const Slack: FunctionComponent<SlackProps> = ({
             <CardContent>
                 <Grid container justifyContent="space-between">
                     <Grid item>
-                        <Typography variant="h6">
-                            <Icon height={12} icon="logos:slack-icon" />{" "}
-                            {t("label.slack")}
-                        </Typography>
+                        <Box clone alignItems="center" display="flex">
```

Comment:
Do we need display to be "flex" here?

Reply:
Yes. Without the flex, the elements are not centered vertically

![Screenshot_134](https://user-images.githubusercontent.com/20799363/220896616-3366b7cd-22d4-4070-8d2e-2e2e37447bce.png)